### PR TITLE
Feature/sc 3199 datasource context menu

### PR DESCRIPTION
**Pull request by adrianjost**
# Issue: [SC-3199](https://ticketsystem.schul-cloud.org/browse/SC-3199)

## Description

- implement context menu in datasource cards on overview page
- let contextmenu handle clicks outside itself to simplify logic
- implement action to disconnect a datasource

## Screenshot

image

### CODE REVIEW

File: locale/de.json [hunk 1]
```diff
@@ -50,6 +50,11 @@
 	"layouts.loggedoutFullWidth.steps": "Erste Schritte",
 	"pages.administration.datasources.index.empty": "Es sind noch keine Datenquellen vorhanden. Mit dem Plus-Button unten rechts kannst du eine neue Datenquelle hinzufügen.",
 	"pages.administration.datasources.index.import": "Daten importieren",
+	"pages.administration.datasources.index.ctxActions.edit": "Zugangsdaten ändern",
+	"pages.administration.datasources.index.ctxActions.remove": "Datenquelle entfernen",
+	"pages.administration.datasources.index.action.remove": "Datenquelle entfernen",
```
Comment (dominikbruechner) [inline locale/de.json]: Same translation? can be merged/reused

File: src/pages/administration/datasources/index.vue [hunk 2]
```diff
@@ -16,9 +16,23 @@
 							<BaseIcon source="custom" icon="datasource-import" />
 							{{ $t("pages.administration.datasources.index.import") }}
 						</BaseButton>
-						<!-- todo use method to return icon -> the method should return the complet icon -->
-						<!-- it exist different sketches with different styles how it should displayed, please ask ui -->
-						<!-- <BaseIcon source="custom" icon="success" :fill="green" /> -->
+						<span style="position: relative">
```
Comment (dominikbruechner) [inline src/pages/administration/datasources/index.vue]: why inline-style?

Comment (adrianjost) [reply src/pages/administration/datasources/index.vue]: agree

File: locale/de.json (hunk 1 above)
Comment (adrianjost) [reply locale/de.json]: true

**Comment by adrianjost** [comment]
@dominikbruechner should be fixed now

**Pull request merged**